rmats.py

```diff
@@ -87,9 +87,15 @@ def doSTARMapping(args): ## do STAR mapping
 
 
 def get_args():
-    """TODO: Docstring for get_args.
-    :returns: TODO
-
+    """Supplies all the neccessary arguments to the argparse package, along with appropriate help, defaults, destinations, and choices.
+    The function itself takes no arguments.
+    Unless rMATS is called in stat mode, exits with appropriate errors if any of: sequence files, gtf, or readlength arguments are missing.
+    If an output directory and/or a temporary directory aren't supplied, exits with appropriate errors.
+    Creates output directory. Cleans user supplied bam or fastq filenames to remove newlines if they follow a space and a comma.
```

Comment:
> remove newlines if they follow a space and a comma

`.strip(' ,\n')` will keep removing characters from the ends of the string until reaching a char that is not in `{' ', ',', '\n'}`. The order of space comma and newline is not important: https://docs.python.org/3/library/stdtypes.html#str.strip

Reply:
ah, thank you. learn something new every day!